react_render/django/render.py

```diff
@@ -60,8 +61,15 @@ def render_component(path_to_source, props=None, to_static_markup=False, json_en
         json_encoder = DjangoJSONEncoder().encode
 
     try:
-        html = render_core(path_to_source, props, to_static_markup, json_encoder, service_url=SERVICE_URL)
-    except:
+        html = render_core(
+            path_to_source,
+            props,
+            to_static_markup,
+            json_encoder,
+            service_url=SERVICE_URL,
+            timeout=TIMEOUT,
```

Comment:
better if this was exposed in `render_component` so it's not just a global timeout.. can choose a timeout based on the component being rendered